packages/loot-design/src/components/modals/CreateLocalAccount.js

```diff
@@ -117,15 +117,31 @@ function CreateLocalAccount({ modalProps, actions, history }) {
                     justifyContent: 'flex-end',
                   }}
                 >
-                  <label style={{ userSelect: 'none' }}>
+                  <label style={{
+                    userSelect: 'none',
+                    textAlign: 'right'
+                  }}>
                     <input
                       name="offbudget"
                       type="checkbox"
                       checked={!!values.offbudget}
                       onChange={handleChange}
                       onBlur={handleBlur}
                     />{' '}
-                    Off-budget
+                    <span> Off-budget </span>
+                    <div style={{ fontSize: '0.7em', color: 'grey' }}>
+                      This cannot be changed later. <br /> See
+                      <a href="https://actualbudget.github.io/docs/Accounts/overview"
+                        target="_blank"
+                        style={{
+                          textDecoration: 'underline',
+                          marginLeft: '2px',
+                          marginRight: '2px'
+                        }}>
```

Comment:
- The underline style is applied by default, so you don’t need to specify it explicitly
- Instead of using pixel margins, you can insert `{' '}` before and after the element to use regular spaces.

Reply:
Also fixed, however I have applied the color to the link for uniformity, hope that's ok, otherwise happy to revert and keep it black.